### DISCUSSION TITLE
Support "2.6 Dependency Patching" of MSVP

### DISCUSSION THREAD

**Discussion by cmlh** [OWASP/ASVS#1692] [Ideas]
["2.6 Dependency Patching" of MVSP](https://github.com/vendorsec/mvsp/blame/master/packages/mvsp/mvsp.en.asciidoc#L90-L91) is reproduced below:

`2.6 Dependency Patching`
` `
`Apply security patches with a severity score of "medium" or higher, or ensure equivalent mitigations are available for all components of the application stack within one month of the patch release
`

The parent of this [MVSP] issue is https://github.com/OWASP/ASVS/issues/1151.

**Comment by cmlh** [comment]
This is out of scope of ASVS which does not verify patching.

**Comment by tghosth** [comment]
Agree
